lib/core/src/model.rs

```diff
@@ -68,53 +68,57 @@ impl Config {
 }
 
 #[derive(Debug, Copy, Clone, PartialEq, Serialize)]
-pub enum Network {
+pub enum LiquidSdkNetwork {
```

Comment:
IMO no need to add "sdk" as we already inside the sdk. LiquidNetwork seems fine.